### PR TITLE
Adding Ashu Kumar and Ravi Waghmare as reviewer.

### PR DESCRIPTION
**Pull request by Ashughorla**
Kindly add Ashu Kumar and Ravi Waghmare as reviewer of HWCC.

@zaneb @dhellmann @digambar15 @fmuyassarov

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -6,3 +6,5 @@ approvers:
  - digambar15
  - russellb
  - maelk
+ - ashughorla
```
Comment (zaneb) [inline OWNERS:11]: This is the approvers list, you need to add a separate list for reviewers. See e.g. https://github.com/metal3-io/baremetal-operator/blob/master/OWNERS for an example.

**Comment by zaneb** [comment]
As discussed in https://kubernetes.slack.com/archives/CHD49TLE7/p1617200202138200

We also need one of the owners to add y'all to the GitHub org before this can merge.
/cc @dhellmann @hardys @stbenjam @russellb

**Comment by dhellmann** [comment]
I will send the GitHub invitations.

**Comment by dhellmann** [comment]
> I will send the GitHub invitations.

Done. @ravipwaghmare and @Ashughorla when you accept the invitations to join the GitHub org, please make the membership public so the bot can see you and give you the permissions you need. Then comment back here to let us know, and we can merge this PR.

File: OWNERS (hunk 1 above)
Comment (Ashughorla) [reply OWNERS:11]: Sure @zaneb, I will raise PR for this also.

**Comment by Ashughorla** [comment]
@dhellmann Accepted invite and made membership public.

File: OWNERS (hunk 1 above)
Comment (Ashughorla) [reply OWNERS:11]: @zaneb, Done. Removed both ID from approvers and added to reviewers.

**Pull request merged**